webserver/static/js/feverscreen.ts

```diff
@@ -1725,37 +1723,39 @@ window.onload = async function () {
     sensorCorrectionDriftOnly: number
   ) {
     drawTargetCircle(hotspot.sensorX, hotspot.sensorY);
```

Comment:
this should probably actually be the face.hotspot, which will be the hottestspot in the haarFace if no forehead is detected, otherwise the hotest spot in the forehead

Reply:
Actually, face.hotspot is what's getting passed to this function.